odoo/modules/registry.py

```diff
@@ -340,17 +321,26 @@ def init_models(self, cr, model_names, context):
                 if name in missing:
                     _logger.info("Recreate table of model %s.", name)
                     env[name].init()
-            cr.commit()
             # check again, and log errors if tables are still missing
             for name, model in env.items():
                 if not model._abstract and not table_exists(cr, model._table):
                     _logger.error("Model %s has no table.", name)
 
+    @lazy_property
+    def cache(self):
+        """ A cache for model methods. """
+        # this lazy_property is automatically reset by lazy_property.reset_all()
+        return LRU(8192)
+
+    def clear_cache(self):
```

Comment:
Having both methods `clear_cache` and `clear_caches` is not ... clear.
This method should be private as it is only intended to be called by `Model.clear_caches` default implementation.